std/wasi/snapshot_preview1_test.ts

```diff
@@ -4,34 +4,88 @@ import { assert, assertEquals } from "../testing/asserts.ts";
 import { copy } from "../fs/mod.ts";
 import * as path from "../path/mod.ts";
 
+const tests = [
+  "testdata/std_env_args_none.wasm",
+  "testdata/std_env_args_some.wasm",
+  "testdata/std_env_vars_none.wasm",
+  "testdata/std_env_vars_some.wasm",
+  "testdata/std_fs_create_dir_absolute.wasm",
+  "testdata/std_fs_create_dir_relative.wasm",
+  "testdata/std_fs_file_create_absolute.wasm",
+  "testdata/std_fs_file_create_relative.wasm",
+  "testdata/std_fs_file_metadata_absolute.wasm",
+  "testdata/std_fs_file_metadata_relative.wasm",
+  "testdata/std_fs_file_seek_absolute.wasm",
+  "testdata/std_fs_file_seek_relative.wasm",
+  "testdata/std_fs_file_set_len_absolute.wasm",
+  "testdata/std_fs_file_set_len_relative.wasm",
+  "testdata/std_fs_file_sync_all_absolute.wasm",
+  "testdata/std_fs_file_sync_all_relative.wasm",
+  "testdata/std_fs_file_sync_data_absolute.wasm",
+  "testdata/std_fs_file_sync_data_relative.wasm",
+  "testdata/std_fs_hard_link_absolute.wasm",
+  "testdata/std_fs_hard_link_relative.wasm",
+  "testdata/std_fs_metadata_absolute.wasm",
+  "testdata/std_fs_metadata_relative.wasm",
+  "testdata/std_fs_read_absolute.wasm",
+  "testdata/std_fs_read_dir_absolute.wasm",
+  "testdata/std_fs_read_dir_relative.wasm",
+  "testdata/std_fs_read_relative.wasm",
+  "testdata/std_fs_remove_dir_all_absolute.wasm",
+  "testdata/std_fs_remove_dir_all_relative.wasm",
+  "testdata/std_fs_rename_absolute.wasm",
+  "testdata/std_fs_rename_relative.wasm",
+  "testdata/std_fs_symlink_metadata_absolute.wasm",
+  "testdata/std_fs_symlink_metadata_relative.wasm",
+  "testdata/std_fs_write_absolute.wasm",
+  "testdata/std_fs_write_relative.wasm",
+  "testdata/std_io_stderr.wasm",
+  "testdata/std_io_stdin.wasm",
+  "testdata/std_io_stdout.wasm",
+  "testdata/std_process_exit.wasm",
+  "testdata/wasi_clock_res_get_monotonic.wasm",
+  "testdata/wasi_clock_res_get_process.wasm",
+  "testdata/wasi_clock_res_get_realtime.wasm",
+  "testdata/wasi_clock_res_get_thread.wasm",
+  "testdata/wasi_clock_time_get_monotonic.wasm",
+  "testdata/wasi_clock_time_get_process.wasm",
+  "testdata/wasi_clock_time_get_realtime.wasm",
+  "testdata/wasi_clock_time_get_thread.wasm",
+  "testdata/wasi_fd_fdstat_get_stderr.wasm",
+  "testdata/wasi_fd_fdstat_get_stdin.wasm",
+  "testdata/wasi_fd_fdstat_get_stdout.wasm",
+  "testdata/wasi_fd_tell_file.wasm",
+  "testdata/wasi_fd_write_file.wasm",
+  "testdata/wasi_fd_write_stderr.wasm",
+  "testdata/wasi_fd_write_stdout.wasm",
+  "testdata/wasi_proc_exit_one.wasm",
+  "testdata/wasi_proc_exit_zero.wasm",
+  "testdata/wasi_random_get.wasm",
+];
```

Comment:
Can't you get this list with readDir?

Reply:
Could just sort the existing readDir list but seems preferable to be explicit here for additional visibility of when tests are changed.

Also, this catches the case of the submodule not being checked out or at an unexpected revision so makes for a slightly happier development path IMHO.